test/lib/xrd.js

```diff
@@ -87,6 +88,8 @@ var xrdLinkCheck = function(def) {
                     }
                 }
             };
+        
+        doc=doc.XRD;
```

Comment:
The update for xml2js, has some consequences. This one sets it right again. The newer xml2js wraps the the object in a XRD object.
```
doc: { name: "John Doe" }
became
doc: { XRD: { name: "John Doe"} }
```